modules/account-lib/src/coin/algo/keyRegistrationBuilder.ts

```diff
@@ -1,26 +1,134 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import BigNumber from 'bignumber.js';
+import algosdk from 'algosdk';
 import { BaseCoin as CoinConfig } from '@bitgo/statics';
-import { NotImplementedError } from '../baseCoin/errors';
-import { BaseKey } from '../baseCoin/iface';
+import { InvalidTransactionError } from '../baseCoin/errors';
 import { TransactionBuilder } from './transactionBuilder';
 import { Transaction } from './transaction';
-
+import { TransactionType } from '../baseCoin';
+import { KeyRegTxnSchema } from './txnSchema';
 export class KeyRegistrationBuilder extends TransactionBuilder {
+  protected _voteKey: string;
+  protected _selectionKey: string;
+  protected _voteFirst: number;
+  protected _voteLast: number;
+  protected _voteKeyDilution: number;
+
   constructor(coinConfig: Readonly<CoinConfig>) {
     super(coinConfig);
   }
+  /**
+   *
```

Comment:
this is a nitpick but we should really be striving to write proper jsdoc documentation here.

https://devhints.io/jsdoc